AppWidget/glance-widget/src/main/java/com/example/android/glancewidget/FirstGlanceWidget.kt

```diff
@@ -47,26 +48,30 @@ class FirstGlanceWidget : GlanceAppWidget() {
         Column(
             modifier = GlanceModifier
                 .fillMaxSize()
-                .background(day = Color.White, night = Color.LightGray)
+                .background(day = Color.White, night = Color.Black)
                 .appWidgetBackground()
                 .cornerRadius(16.dp)
-                .padding(8.dp)
+                .padding(16.dp)
         ) {
             Text(
                 text = "First Glance widget",
-                modifier = GlanceModifier.fillMaxWidth(),
+                modifier = GlanceModifier
+                    .fillMaxWidth()
+                    .padding(bottom = 8.dp),
                 style = TextStyle(fontWeight = FontWeight.Bold),
             )
             Row(
                 modifier = GlanceModifier.fillMaxSize(),
                 verticalAlignment = Alignment.CenterVertically
             ) {
                 Button(
-                    text = "button1",
+                    text = "Button 1",
+                    modifier = GlanceModifier.height(48.dp),
```

Comment:
Is the default hight lower than 48? By default the button should have a proper height

Reply:
Looks like the Button's height doesn't have a proper minimum height, I'll check the code of the Glance.